02-analytics/technical.md

```diff
@@ -0,0 +1,393 @@
+
+The data specified below should be sent with every event, unless otherwise noted. 
+If it is not possible for all events then that should be noted in this document.
+
+In general it is better to omit data (if for example it is not accessible on some
+os version), and send what you can.
+
+Note that this is for both APP and Web so some things may be possible on one 
+platform and not on the other.
+
+### Common data table
+
+| Data                | Name           | Comments                                      |
+|---------------------|----------------|-----------------------------------------------|
+| OS                  | os             | Automatically managed by SDKs                 |
+| OS Language         | locale         | Automatically managed by SDKs                 |
+| Network Info        | networkInfo    | Automatically managed by SDKs                 |
+| Device Info         | deviceInfo     | Automatically managed by SDKs                 |
+| Anonymous ID        | anonymousId    | Automatically managed by SDKs                 |
+| Channel             | channel        | Automatically managed by SDKs                 |
+| Timezone            | timezone       | Automatically managed by SDKs                 |
+| Screen data         | screen         | Automatically managed by SDKs                 |
+| App Language        | appLanguage    |                                               |
+| Release Version     | releaseVersion | App version, or build number/git hash for web |
+| Person analytics ID | presonId       | For logged in users, see below                |
+
+### Person analytics ID
+
+You can get this ID by sending a token to TODO. The returned value is an hash
+of the userID + a secret. This ensures that we are able to link all data to one
+user from different systems, while preventing anyone from being able to reverse
+the process or generate analytics IDs if they do not posses the secret.
+
+## Identify
+
+*When*: On login + When you request a new refresh token from Auth0
+*Reason*: Being able to connect anonymous events to a user
+
+### API
+
+Use the `/identify` endpoint. Docs:			https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/identify
+
+### Data
+
+This call accepts much more data, but we do not want to send it here. The additional
+data will be automatically injected by RudderStack, and delivered to the needed targets.
+
+| Data         | Name        | Comments                                                                         |
+|--------------|-------------|----------------------------------------------------------------------------------|
+| Anonymous ID | anonymousId | Automatically managed by SDKs                                                    |
+| PersonID     | id          | This is the actual person ID. It will be used for data lookup and then discarded |
+
+## Screen/Page View
+
+*When*: On every page/screen the user opens. If relevant also on popups if they
```

Comment:
Are changing tabs on live page or audience page are also considered as separate screen views?

Reply:
Yes, they should be.